plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/DriverDependenciesTree.java

```
@@ -175,6 +178,16 @@ public boolean loadLibDependencies() throws DBException {
         return resolved;
     }
 
+    private void grayOutInstalledArtifact(DBPDriverDependencies.DependencyNode node, TreeItem item) {
+        Path localFile = node.library.getLocalFile();
+        try {
+            if ((localFile != null && Files.exists(localFile) && Files.size(localFile) > 0) && editable) {
+                item.setForeground(filesTree.getDisplay().getSystemColor(SWT.COLOR_WIDGET_DARK_SHADOW));
+            }
+        } catch (IOException ignored) {
```

Never ignore exceptions